src/bindings/python/src/pyopenvino/graph/model.cpp

```diff
@@ -701,9 +731,87 @@ void regclass_graph_Model(py::module m) {
         return "<" + class_name + ": '" + self.get_friendly_name() + "'\ninputs[\n" + inputs_str + "\n]\noutputs[\n" +
                outputs_str + "\n]>";
     });
+    model.def("get_rt_info",
+              (PyRTMap & (ov::Model::*)()) & ov::Model::get_rt_info,
+              py::return_value_policy::reference_internal,
+              R"(
+                Returns PyRTMap which is a dictionary of user defined runtime info.
+
+                :return: A dictionary of user defined data.
+                :rtype: openvino.runtime.RTMap
+             )");
+    model.def(
+        "get_rt_info",
+        [](const ov::Model& self, const py::list& args) -> py::object {
+            const size_t args_len = py::len(args);
+            std::vector<std::string> cpp_args;
+            for (size_t i = 0; i < args_len; i++) {
+                cpp_args.emplace_back(args[i].cast<std::string>());
+            }
+            return Common::utils::from_ov_any(
+                ov::get_rt_info(self, self.get_rt_info(), cpp_args.cbegin(), cpp_args.cend()));
+        },
+        py::arg("args") = py::list(),
+        R"(
+                Returns runtime attribute.
+
+                :param args: List of strings which defines a path for rt info
+                :type args: List[str]
+
+                :return: A py::object
+             )");
+    model.def(
+        "has_rt_info",
+        [](const ov::Model& self, const py::list& args) -> bool {
```

Comment:
```suggestion
        [](const ov::Model& self, const py::list& path) -> bool {  // path?
```
This is a comment about all `args` usages.

I wouldn't call `args` a restricted name by default. However, in Python `args` name is very often used to describe arbitrary number of unnamed arguments (something like variadic templates). So in this case, some user might expect this function to accept `model.has_rt_info("MO", 1, [2,3,4])` or something similar.

I would change all `args` to something more descriptive like `path` or `path_to_attr`. It is open discussion on the name, but still it should be changed to avoid any confusion.

Ref: https://pybind11.readthedocs.io/en/stable/advanced/functions.html#accepting-args-and-kwargs